src/components/Stack/README.md

```diff
@@ -87,7 +87,7 @@ Use to control spacing of items in a stack in standard increments. Use tight for
 Use to vertically center a set of items.
 
 ```jsx
-<Stack alignment="center">
+<Stack vertical alignment="center">
```

Comment:
I don't think that's the intention here. This uses the `align-items` flex attribute.

![image](https://user-images.githubusercontent.com/6239480/51918236-ffc96a00-2395-11e9-9e3b-bfb727076846.png)

Reply:
So, it's not an example of vertically stacking items, rather _centering_ them along a vertical axis.